app/src/main/java/com/example/dogbook/MainActivity/MainActivity.java

```diff
@@ -1,16 +1,72 @@
 package com.example.dogbook.MainActivity;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
 
+import com.example.dogbook.LoginActivity.LoginActivity;
+import com.example.dogbook.MainActivity.Fragments.TimelineFragment;
 import com.example.dogbook.R;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+import com.parse.ParseUser;
 
 public class MainActivity extends AppCompatActivity {
 
+    private Toolbar toolbar;
+    private BottomNavigationView navigationBar;
+    private FragmentManager fragmentManager = getSupportFragmentManager();
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        initView();
+
+        navigationBar.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(MenuItem item) {
+                Fragment fragment = new TimelineFragment();
+                fragmentManager.beginTransaction().replace(R.id.fragmentContainer, fragment).commit();
+                return true;
+            }
+        });
+
+        navigationBar.setSelectedItemId(R.id.item_home); //Display home when initialized
```

Comment:
This seems like part of initializing the view -- you're setting up the navigation bar items here.  I would move it into the initialization method